fearless/Common/Services/GitHubPhishingService/PhishingCheckExecutor.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import RobinHood
+import CoreData
+import SoraFoundation
+import CommonWallet
+
+protocol PhishingCheckExecutorProtocol {
+    func checkPhishing(publicKey: String, walletAddress displayName: String)
+}
+
+class PhishingCheckExecutor<T: Identifiable, U: NSManagedObject>: PhishingCheckExecutorProtocol {
+    private var storage: CoreDataRepository<T, U>
```

Comment:
It is more convenient to use AnyDataProviderRepository. In this case one doesn't care about database type